docs/atomic-sign.1.md

```diff
@@ -0,0 +1,43 @@
+% ATOMIC(1) Atomic Man Pages
+% Brent Baude
+% August 2016
+# NAME
+atomic-sign- Create a signature for a local image
+# SYNOPSIS
+**atomic sign**
+[**-h**|**--help**]
+
+[**-o**, **--output**]
+[**--sign_key**]
+[ image ... ]
+
+# DESCRIPTION
+**atomic sign** will create a local signature for one or more local images that have 
+been pulled from a registry. Unless overridden, the signature will end up in a directory
+as follows:
+
+ /etc/pki/containers/registry_name/image_name@sha256:image_id/
+
+The signatures are named 'signature-INTEGER' where the integer is increased by 1 each
+time.
+
+# OPTIONS
+**-h** **--help**
+  Print usage statement.
+
+**-o** **--output**
+  Used to override the default signature location.  It must be a fully qualified 
+  path name (including file name) for where to store the signature.
+
+**--sign_key**
+  Override the default signature and use the defined signature instead.
+
```

Comment:
Added location information and information around defaults.  Added two additional signing examples.